xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailMetaSheetView.swift

```diff
@@ -82,6 +74,22 @@ struct MemoViewerDetailMetaSheetModel: ModelProtocol {
     var noteVersion: String?
     var authorKey: String?
     
+    var shareableLink: String? {
+        guard let address = address else {
+            Self.logger.log("Copy link: (nil)")
```

Comment:
We should remove the logging. I had this in the update function since clipboard seemed like a relevant event. However share link does not do the same modification of global persistent state.

Reply:
Done!